src/gsy_e/models/leaves.py

```diff
@@ -76,7 +79,8 @@ class LeafBase:
     strategy = None
 
     def __init__(self, name, config, uuid=None, **kwargs):
-        if config.external_connection_enabled:
+        if (config.external_connection_enabled and
+                ConstSettings.MASettings.MARKET_TYPE != SpotMarketTypeEnum.COEFFICIENTS.value):
```

Comment:
This is not part of the fix, only to remove the error messages that appear when launching SCMs